### ISSUE TITLE
cannot import from undefined module: srfi-132

### ISSUE THREAD

**Issue by diamond-lizard** [diamond-lizard/srfi-132#2]
When running ```test-new-egg``` on the code that is currently tagged [0.1-pre1](https://github.com/diamond-lizard/srfi-132/releases/tag/0.1-pre1), there's an error:
```
Error: (import) during expansion of (import ...) - cannot import from undefined module: srfi-132

	Call history:

	<syntax>	  (##core#quote expand)
	<eval>	  (##sys#load-library (##core#quote library))
	<eval>	  (##sys#load-library (##core#quote library))
	<eval>	  (##sys#load-library (##core#quote expand))
	<syntax>	  (import-syntax chicken.base chicken.fixnum chicken.platform (except scheme string-copy string-fill! ...
	<syntax>	  (##core#undefined)
	<syntax>	  (##core#undefined)
	<syntax>	  (##core#undefined)
	<syntax>	  (##core#begin (##core#require r7rs r7rs#))
	<syntax>	  (##core#require r7rs r7rs#)
	<syntax>	  (chicken.load#load-extension (##core#quote r7rs) (##core#quote (r7rs#)) (##core#quote require))
	<syntax>	  (##core#quote r7rs)
	<syntax>	  (##core#quote (r7rs#))
	<syntax>	  (##core#quote require)
	<eval>	  (chicken.load#load-extension (##core#quote r7rs) (##core#quote (r7rs#)) (##core#quote require))
	<syntax>	  (import srfi-132)	<--
```

A full log can be found [here](https://github.com/diamond-lizard/srfi-132/files/5576825/srfi-132-error.txt)

**Comment by diamond-lizard** [comment]
On #chicken, jcowan said:
```
Sounds like you are not generating srfi-132.import.scm
```

Indeed, looking at the contents of the [salmonella.log](https://github.com/diamond-lizard/srfi-132/files/5576825/srfi-132-error.txt) shows that ```srfi-132.import.scm``` is not mentioned.

However, it was generated by ```chicken-install -test```

So I'm not sure why it's being generated by ```chicken-install -test``` but not by ```test-new-egg```

Nor do I know how to fix it.

**Comment by diamond-lizard** [comment]
In #chicken, jcowan suggested:
```
csc -P -J srfi-132.scm
```

**Comment by diamond-lizard** [comment]
Fixed in commits 1fbedd8 and 054656e
